hpedockerplugin/volume_manager.py

```diff
@@ -447,6 +446,8 @@ def manage_existing(self, volname, existing_ref, backend='DEFAULT'):
                 self.map_3par_volume_prov_to_docker(volume_detail_3par)
             vol['compression'] = \
                 self.map_3par_volume_compression_to_docker(volume_detail_3par)
+            vol['cpg'] = volume_detail_3par.get('userCPG')
```

Comment:
In case of snapshots, userCPG field will not be available. Hence, I feel this will throw an error. Can you please recheck.

Reply:
In that case, it will return None.

Reply:
Yah, corret!